lib/datadog/tracing/contrib/http/circuit_breaker.rb

```diff
@@ -29,6 +29,14 @@ def internal_request?(request)
           end
 
           def should_skip_distributed_tracing?(client_config)
+            if Datadog.configuration.appsec.standalone.enabled
+              active_trace = Tracing.active_trace
+
+              # return true if absence of upstream ASM event (_dd.p.appsec:1)
+              # and no local security event (which sets _dd.p.appsec:1 locally)
+              return true if active_trace.nil? || active_trace.get_tag(Datadog::AppSec::Ext::TAG_APPSEC_EVENT) != '1'
```

Comment:
Will it be possible to cut 1 namespace? (not sure)

```suggestion
              return true if active_trace.nil? || active_trace.get_tag(AppSec::Ext::TAG_APPSEC_EVENT) != '1'
```

Reply:
It does work but it seems that there's a convention in the codebase to still add the parent namespace (I suppose for readability). You can see another example at l.27-28 of the same file.